ingestion/src/metadata/ingestion/source/glue.py

```diff
@@ -0,0 +1,208 @@
+import json
+import logging
+import os
+import traceback
+import uuid
+from typing import Iterable, Optional
+
+import boto3
+
+from ...generated.schema.api.services.createPipelineService import (
+    CreatePipelineServiceEntityRequest,
+)
+from ...generated.schema.entity.data.database import Database
+from ...generated.schema.entity.data.pipeline import Pipeline, Task
+from ...generated.schema.entity.data.table import Column, Table
+from ...generated.schema.entity.services.databaseService import DatabaseServiceType
+from ...generated.schema.type.entityReference import EntityReference
+from ...utils.column_helpers import check_column_complex_type
+from ...utils.helpers import get_database_service_or_create
+from ..api.common import ConfigModel, IncludeFilterPattern, Record
+from ..api.source import Source, SourceStatus
+from ..models.ometa_table_db import OMetaDatabaseAndTable
+from ..ometa.ometa_api import OpenMetadata
+from ..ometa.openmetadata_rest import MetadataServerConfig
+from ..source.sample_data import get_pipeline_service_or_create
+from .sql_source import SQLConnectionConfig, SQLSourceStatus
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+class GlueSourceConfig(ConfigModel):
+    service_type = "MySQL"
```

Comment:
we should add "Glue" as the service here

Reply:
Changed it